### PR TITLE
CFE-3812: cfbs init now prompts user whether to use default masterfiles

### PR DESCRIPTION
**Pull request by larsewi**
The cfbs init command now prompts users whether or not they want to use
the default masterfiles policy framework. If the user says no, they're
prompted with which masterfiles they want to add, allowing them to skip
if empty.

Ticket: CFE-3812
Changelog: Title
Signed-off-by: Lars Erik Wik <lars.erik.wik@northern.tech>

### CODE REVIEW

**Review by vpodzime — COMMENTED** [review]
So `cfbs --non-interactive init` now adds MPF? Is that what we want?

File: cfbs/commands.py [hunk 1]
```diff
@@ -193,7 +193,26 @@ def init_command(index=None, non_interactive=False) -> int:
...
+    CFBSConfig.instance = None
+
+    if prompt_user(
+        "Do you wish to add default masterfiles?",
+        choices=YES_NO_CHOICES,
+        default="yes",
+    ) in ("yes", "y"):
+        to_add = "masterfiles"
+    else:
+        to_add = prompt_user("Specify masterfiles module or skip if empty", default="")
```
Comment (larsewi) [inline cfbs/commands.py:212]:
```suggestion
        to_add = prompt_user("Specify policy set to use instead (or skip if empty)", default="")
```

**Comment by larsewi** [comment]
> So `cfbs --non-interactive init` now adds MPF? Is that what we want?

The option `--non-interactive` should probably perform the default behaviour?  In this case the question would be; should `cfbs init` add MPF by default? 🤔  What do you think @vpodzime @olehermanse?

**Comment by olehermanse** [comment]
> So `cfbs --non-interactive init` now adds MPF? Is that what we want?

Yes

**Comment by olehermanse** [comment]
> > So `cfbs --non-interactive init` now adds MPF? Is that what we want?
> 
> 
> 
> The option `--non-interactive` should probably perform the default behaviour?  In this case the question would be; should `cfbs init` add MPF by default? 🤔

Yes and yes

**Review by olehermanse — CHANGES_REQUESTED** [review]

**Review by olehermanse — APPROVED** [review]

**Pull request merged**